testing/trino-testing/src/main/java/io/trino/testing/BaseFailureRecoveryTest.java

```diff
@@ -103,6 +103,7 @@ protected final QueryRunner createQueryRunner()
                         .put("retry-policy", retryPolicy.toString())
                         .put("retry-initial-delay", "0s")
                         .put("retry-attempts", "1")
+                        .put("query.max-memory-per-task", "128MB")
```

Comment:
Is it still needed?

Reply:
Nope - actually tests failed because of that :)